CSS/CSS.sublime-syntax

```diff
@@ -6,6 +6,24 @@ file_extensions:
   - css
   - css.erb
 scope: source.css
+variables:
+  custom_element_chars: '(?x)
```

Comment:
Don't do this. If you want to use multi-line extended regular expressions in variables, **always** wrap the entire content in `(?x: ... )` and ensure there is no trailing whitespace at the end of the YAML string. This will ensure that inclusion of the variable won't have any side effects on the actual pattern. Check out [YAML.sublime-syntax](https://github.com/sublimehq/Packages/blob/master/YAML/YAML.sublime-syntax#L58-L69) for examples.

In addition to everey whitespace following the variable insertion becoming insignificant, you also add pipe junctions in the level the variable was included it. This can be worked around by always wrapping the variable inside some group, capturing or non-capturing, but imo it's good practice to always store and use "normal" variables as atomic, unless specifically prefixed/suffixed.